pages/burials-and-memorials/pre-need/form-10007-apply-for-eligibility.md

```diff
@@ -1,15 +0,0 @@
----
-title: Apply online for pre-need determination of eligibility in a VA National Cemetery
-display_title: Apply Now
-entryname: pre-need
-layout: page-react.html
-order: 1
----
-<nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs"
-id="va-breadcrumbs">
-  <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
-    <li><a href="/">Home</a></li>
-    <li><a href="/burials-memorials/">Burials and Memorials</a></li>
-    <li><a aria-current="page" href="/burials-memorials/pre-need-eligibility/">Pre-need Eligibility Determination</a></li>
-  </ul>
-</nav>
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-ef0b50656d1d157b0c7b03f406b5f2fbR5.

Breadcrumbs confirmed to look okay.